deepchem/utils/evaluate.py

```diff
@@ -287,6 +287,17 @@ def compute_model_performance(
       If `per_task_metrics == True`, then returns a second dictionary
       of scores for each task separately.
     """
+    if n_classes is not None:
+      logger.warning(
+          "n_classes is deprecated as an argument and will be removed in a future version of DeepChem."
+          "In future version, n_classes will be directly inferred from the model"
+      )
+    if hasattr(self.model, 'n_classes'):
+      if n_classes != self.model.n_classes:
+        logger.info(
```

Comment:
I would make this `logger.warning`.  If we're ignoring the argument they passed, that's more than just an info.

Reply:
I updated with the same.